CMakeLists.txt

```diff
@@ -351,7 +351,7 @@ else()
 
     # add the non-optional SFML headers
     add_custom_command(TARGET SFML POST_BUILD COMMAND cp -r
-                       ${PROJECT_SOURCE_DIR}/include/SFML/Config.hpp
+                       ${SFML_BINARY_DIR}/configured/include/SFML/Config.hpp
```

Comment:
Can you use `PROJECT_BINARY_DIR` to better match existing style since `SFML_BINARY_DIR` is not currently used?